python-package/pyproject.toml

```diff
@@ -26,6 +26,15 @@ readme = "README.rst"
 requires-python = ">=3.6"
 version = "3.3.5.99"
 
+[project.optional-dependencies]
+dask = [
+    "dask[array,dataframe,distributed]>=2.0.0",
```

Comment:
The format where these are split out as individual items

```text
dask[array]>=2.0.0
dask[dataframe]>=2.0.0
dask[distributed]>=2.0.0
```

Is confusing. It makes it seem like 3 separate versions of 3 separate things, which just happen to be the same... but that's not really what it means.

That is 3 different ways to say `dask >= 2.0.0`.

We should just use a single number, to make that clearer.